lib/xandra/cluster.ex

```diff
@@ -194,10 +200,16 @@ defmodule Xandra.Cluster do
             "the :priority load balancing strategy is only supported when :autodiscovery is false"
     end
 
+    if !autodiscovery? and load_balancing == :dc_aware do
```

Comment:
You can use `not/1` for strictly booleans:

```suggestion
    if not autodiscovery? and load_balancing == :dc_aware do
```